### ISSUE TITLE
chrome-debug command not working in v9.6.2+

### ISSUE THREAD

**Issue by CalumY** [GoogleChrome/lighthouse#14222]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### URL

https://example.com/

### What happened?

When trying to run `chrome-debug`, the error message in the "Relevant log output" section below was shown. 

### What did you expect?

I expected an instance of chrome to be launched and a message showing the debugging port to be printed.

### What have you tried?

I tried installing lighthouse v9.6.1, and this issue did not occur. This error message only shows in v9.6.2 and v9.6.3.

### How were you running Lighthouse?

CLI

### Lighthouse Version

v9.6.2

### Chrome Version

103.0.5060.114

### Node Version

v16.16.0

### OS

Windows

### Relevant log output

```shell
import {Launcher, launch} from 'chrome-launcher';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1033:15)
    at Module._compile (node:internal/modules/cjs/loader:1069:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47
```

**Comment by adamraine** [comment]
I can reproduce this but only on windows, looks like an esm issue

**Comment by adamraine** [comment]
This package.json wasn't released with v9.6.2 or v9.6.3 for some reason:
https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/scripts/package.json

**Comment by adamraine** [comment]
Actually I can reproduce on MacOS now too
